src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -35,6 +38,38 @@ public void mercury_search_navigateToPageUsingSearchSuggestions() {
     Assertion.assertTrue(driver.getCurrentUrl().equals(suggestionLink));
   }
 
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnMobile")
+  public void mercury_search_navigateUsingSearchSuggestionsOnMobile() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)